pkg/cloud/cloud.go

```diff
@@ -280,29 +289,27 @@ func NewCloud(region string, awsSdkDebugLog bool, userAgentExtra string, batchin
 }
 
 func newEC2Cloud(region string, awsSdkDebugLog bool, userAgentExtra string, batchingEnabled bool) Cloud {
-	awsConfig := &aws.Config{
-		Region:                        aws.String(region),
-		CredentialsChainVerboseErrors: aws.Bool(true),
-		// Set MaxRetries to a high value. It will be "overwritten" if context deadline comes sooner.
-		MaxRetries: aws.Int(8),
+	cfg, err := config.LoadDefaultConfig(context.TODO(), config.WithRegion(region))
```

Comment:
Nit: It may technically be out of scope for this PR, but I don't think we shouldhave `context.TODO()`here  in our next release (perhaps a high priority backlog item?)

And maybe we can rely on this future context as the timeout for other driver-startup tasks such as taint removal? To have one source of truth for `driver startup is taking too long, lets restart the container` rather than our amalgamation of different timeouts?

Reply:
Switched from `context.TODO()` to `context.Background()` given a timeout here is not incredibly relevant since any failures at this point will be handled by the livenessprobe.